libs/blocks/merch-card/merch-card.js

```diff
@@ -201,6 +201,35 @@ const parseContent = async (el, merchCard) => {
       }
       return;
     }
+    if (tagName === 'H6' && element.firstElementChild?.tagName === 'EM') {
+      const calloutContentWrapper = createTag('div');
+      const calloutContent = createTag('div');
+      const emElement = element.firstElementChild;
+      let imgElement = null;
+      const children = Array.from(emElement.childNodes);
+      for (let i = 0; i < children.length; i += 1) {
+        const child = children[i];
+        if (child.nodeType === Node.ELEMENT_NODE && child.tagName === 'A' && child.innerText.trim() === '#ICON') {
+          const [imgSrc, tooltipText] = child.getAttribute('href')?.split('#') || [];
+          imgElement = document.createElement('img');
+          imgElement.src = imgSrc;
+          imgElement.title = decodeURIComponent(tooltipText);
+          imgElement.className = 'callout-icon';
+          child.parentNode.removeChild(child);
+          calloutContentWrapper.classList.add('callout-content-wrapper-with-icon');
```

Comment:
wouldn't it make more sense to add the class when the image element is actually appended?